pkgs/applications/science/logic/tlaplus/tla-toolbox.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Exec=tla-toolbox
+Icon=tla-toolbox
+Comment=IDE for TLA+
+Terminal=false
+Name=TLA Toolbox
+Categories=Application;Development;
+StartupWMClass=TLA+ Toolbox
```

Comment:
For desktop items we usually use this: 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/browsers/chromium/default.nix#L37

Reply:
ooh that's useful. I'll do that. Thanks for the pointer!